ua-snap/fire-tally
# Adds date range selectors for the other 2 charts

* Also corrects an issue in the source data file
 * Improves chart titles

To test, check this branch out and ensure that:
 * changing the date range selectors for all 3 charts works as expected
 * chart title change to match 
 * changing other parameters (protection zone or year) maintains that chart's date range selection , and still responds to changing the date range selection.
 * (data test) there is no huge doubling/bump in the "select by protection area" or "select by year" charts.  (Removed doubled data for that day in source file.)
